### PR TITLE
[WIP] make goby support syntactic sugar

### PR DESCRIPTION
**Pull request by HexterCH**
[Issues 241](https://github.com/goby-lang/goby/issues/241)

### CODE REVIEW

File: compiler/token/token.go [hunk 1]
```diff
@@ -24,7 +24,9 @@ const (
 
 	Assign   = "="
 	Plus     = "+"
+	PlusShorthand = "+="
```
Comment (st0012) [inline compiler/token/token.go]: I think `PlusEql` might be simpler?

**Comment by st0012** [comment]
@HexterCH I don't want to push you but as @Maxwell-Alexius keep developing http library he will need these syntactic sugars.

**Comment by HexterCH** [comment]
I need some help, I am stuck now.

**Comment by st0012** [comment]
What?

**Comment by st0012** [comment]
@HexterCH This feature has been implemented, so I'm closing it now~

**Pull request closed without merging**